contracts/src/v0.8/operatorforwarder/test/Broken.sol

```diff
@@ -1,6 +1,7 @@
 pragma solidity ^0.8.0;
 
 // Broken is a contract to aid debugging and testing reverting calls during development.
+// solhint-disable
 contract Broken {
```

Comment:
This project is super old, not even sure if its maintained or used tbh. I wonder if it can be axed in a future PR?

Reply:
yeah this PR moves stuff thats used into project folders. I great follow up is to ask each project team to go through their own folder and delete unused files!